### PR TITLE
Dashboard

### PR DESCRIPTION
**Pull request by GeneralSnuggles**

### CODE REVIEW

**Comment by maziland** [comment]
@GeneralSnuggles is it working?

File: services/fleet.js [hunk 1]
```diff
@@ -20,6 +20,30 @@ async function fleetApiGetRequest(uri) {
...
+    return data;
+};
+
+async function getScriptByEndpoint(endpointList){
+    // put in loop over endpointList
+    // change id to endpoint id
+    let scriptUri = 'GET /api/v1/fleet/hosts/:id/scripts'
+    // merge all api answers
+    // return all the data
+};
```
Comment (maziland) [inline services/fleet.js]: This is not finished yet right?

Comment (GeneralSnuggles) [reply services/fleet.js]: Yes, template before adding the actual code

File: views/dashboard.ejs [hunk 2]
```diff
@@ -7,29 +7,71 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Dashboard</title>
+    <style>
+        .status-error { color: red; }
+        .status-pending { color: orange; }
+        .status-ran { color: green; }
+    </style>
```
Comment (maziland) [inline views/dashboard.ejs]: Put these lines on `dashboard.css`

**Review by maziland — APPROVED** [review]
Looks good, just move the style block to the css file

File: views/dashboard.ejs (hunk 2 above)
Comment (GeneralSnuggles) [reply views/dashboard.ejs]: done

**Pull request merged**